routes/tripRouter.js

```diff
@@ -16,6 +16,22 @@ tripRouter.post('/', (req, res, next) => {
         })
 });
 
+tripRouter.get('/init/:id', (req, res, next) => {
+    const { id } = req.params;
+    // This route retrieves a trip's basic bag and todolist info
+    const tripDetails = TripService.read(id)
+    const allLists = TripService.getAllListsByTripID(id)
+    const allBags = TripService.getAllBagsByTripID(id);
+    Promise.all([tripDetails, allLists, allBags])
```

Comment:
There are some usages of `async`/`await` elsewhere in your GitHub activity. Would you be up for switching this to be an `async` function as well? IMO that'd be a little more readable.

Instead of something like this:

```js
function example() {
    Promise.all([getOne, getTwo])
        .then(([one, two]) => { /* ... */ })
        .catch(error => { /* ... */ });
}
```

You could do something like:

```js
async function example() {
    try {
        const [one, two] = await Promise.all([getOne, getTwo]);
        /* ... */
    } catch (error) { /* ... */ }
}
```

Not a blocker at all 😄 just a stylistic suggestion.

Reply:
Definitely will do this, I haven't really dabbled with async/await so this will be good practice.